src/client/v2/algod/models/types.ts

```diff
@@ -1106,7 +1119,10 @@ export class ApplicationParams extends BaseModel {
         ? base64ToBytes(clearStateProgram)
         : clearStateProgram;
     this.creator = creator;
-    this.extraProgramPages = extraProgramPages;
+    this.extraProgramPages =
+      typeof extraProgramPages === 'undefined'
+        ? undefined
+        : ensureSafeInteger(extraProgramPages);
```

Comment:
Why wouldn't we want to coerce to 0 instead of `undefined`?  The question probably comes from me not having enough context to know where this type is used.